roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: st2
+    baseurl: https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/el/{{ ansible_distribution_major_version }}/$basearch
+    # TODO(mierdin): Should consider moving back to $releasever once https://github.com/StackStorm/ansible-st2/issues/89 is addressed
```

Comment:
`TODO` line can be removed as well, since you fixed https://github.com/StackStorm/ansible-st2/issues/89